ferrocene/doc/user-manual/src/self-test/error-codes.rst

```diff
@@ -71,16 +70,16 @@ sysroot.
 FST_007: Binary version mismatch
 --------------------------------
 
-This error occurs when the version of the ``ferrocene-self-test`` binary and
-that of the Ferrocene compiler are not the same.
+This error occurs when the versions of the ``ferrocene-self-test`` binary and
+that of the rustc compiler are not the same.
```

Comment:
you can either say *the Rust compiler* or *rustc*, generally
```suggestion
that of ``rustc`` are not the same.
```

Reply:
Can do that. I'd go for rustc.